SilKit/source/services/logging/Logger.cpp

```diff
@@ -101,11 +281,26 @@ Logger::Logger(const std::string& participantName, Config::Logging config)
     localtime_r(&timeNow, &tmBuffer);
 #endif
 
+    // Defined JSON pattern for the logger output
+    std::string jsonpattern  {R"({"ts":"%Y-%m-%dT%H:%M:%S.%e%z","log":"%n","lvl":"%l", %v })"}; 
```

Comment:
I think we should increase the timestamp resolution to at least microseconds. Especially trace-level logs are created with sub-millisecond deltas.

`spdlog` should support a `%f` (instead of `%e`). It might be debatable if we need nanosecond resolution.

I could also see a format of `<full-seconds-since-epoch>.<subsecond-remainder>` (with `subsecond` being either `microsecond` or `nanosecond`), but there seems to be no good way to do that in `spdlog` (or C++14 without third-party libraries for that matter) with respect to the UNIX epoch.

Could you ask this in the next daily, there could be different opionions to consider.